scipy/stats/tests/test_bootstrap.py

```diff
@@ -196,6 +201,54 @@ def my_stat(data1, data2, axis=-1):
     assert pvalue > 0.1
 
 
+
+@pytest.mark.parametrize("method", ["basic", "percentile"])
+@pytest.mark.parametrize("axis", [0, 1])
+def test_bootstrap_vectorized_3samp(method, axis):
+    def statistic(*data, axis=0):
+        # an arbitrary, vectorized statistic
+        return sum((sample.mean(axis) for sample in data))
+
+    def statistic_1d(*data):
+        # the same statistic, not vectorized
+        for sample in data:
+            assert sample.ndim == 1
+        return statistic(*data, axis=0)
+
+    np.random.seed(0)
+    x = np.random.rand(4, 5)
+    y = np.random.rand(4, 5)
+    z = np.random.rand(4, 5)
```

Comment:
I prefer to use `random` over `rand`. To me, NumPy's doc suggest that this might be removed at some point as it points to `random_sample` which points to `random`. But as you wish.
```suggestion
    x = np.random.random(4, 5)
    y = np.random.random(4, 5)
    z = np.random.random(4, 5)
```

Reply:
I came from Matlab, and `rand` is not deprecated, so I use `rand`.